tools/lib/h5tools.h

```diff
@@ -227,67 +229,82 @@ typedef struct h5tool_format_t {
      *
-     *   fmt_ulong:  The printf() format to use when rendering data which is
-     *               typed `unsigned long'. The default is `%lu'.
+     *   fmt_ulong:            The printf() format to use when rendering data which is
+     *                         typed `unsigned long'. The default is `%lu'.
      *
-     *   fmt_llong:  The printf() format to use when rendering data which is
-     *               typed `long long'. The default depends on what printf()
-     *               format is available to print this datatype.
+     *   fmt_llong:            The printf() format to use when rendering data which is
+     *                         typed `long long'. The default depends on what printf()
+     *                         format is available to print this datatype.
      *
-     *   fmt_ullong: The printf() format to use when rendering data which is
-     *               typed `unsigned long long'. The default depends on what
-     *               printf() format is available to print this datatype.
+     *   fmt_ullong:           The printf() format to use when rendering data which is
+     *                         typed `unsigned long long'. The default depends on what
+     *                         printf() format is available to print this datatype.
      *
-     *   fmt_double: The printf() format to use when rendering data which is
-     *               typed `double'. The default is `%g'.
+     *   fmt_ldouble:          The printf() format to use when rendering data which is
+     *                         typed `long double'. The default is `%Lg'.
      *
-     *   fmt_float:  The printf() format to use when rendering data which is
-     *               typed `float'. The default is `%g'.
+     *   fmt_double:           The printf() format to use when rendering data which is
+     *                         typed `double'. The default is `%g'.
      *
-     *   ascii:      If set then print 1-byte integer values as an ASCII
-     *               character (no quotes).  If the character is one of the
-     *               standard C escapes then print the escaped version.  If
-     *               the character is unprintable then print a 3-digit octal
-     *               escape.  If `ascii' is zero then then 1-byte integers are
-     *               printed as numeric values.  The default is zero.
+     *   fmt_float:            The printf() format to use when rendering data which is
+     *                         typed `float'. The default is `%g'.
      *
-     *   str_locale: Determines how strings are printed. If zero then strings
-     *               are printed like in C except. If set to ESCAPE_HTML then
-     *               strings are printed using HTML encoding where each
-     *               character not in the class [a-zA-Z0-9] is substituted
-     *               with `%XX' where `X' is a hexadecimal digit.
+     *   fmt_float_complex:    The printf() format to use when rendering data which is
```

Comment:
Should probably follow the same (big->little) ordering as for floating-point types.

Reply:
Or switch the floating-point types to be little->big also.